ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessStatus.scala

```diff
@@ -6,25 +6,32 @@ import io.circe.generic.JsonCodec
 import io.circe.{Decoder, Encoder, Json}
 import pl.touk.nussknacker.engine.api.deployment.ProcessActionType.ProcessActionType
 import pl.touk.nussknacker.engine.api.deployment.simple.{SimpleProcessStateDefinitionManager, SimpleStateStatus}
-import pl.touk.nussknacker.engine.api.deployment.{ProcessState, ProcessStateDefinitionManager, StateStatus}
+import pl.touk.nussknacker.engine.api.deployment.{CustomAction, ProcessState, ProcessStateDefinitionManager, StateStatus}
 
 //TODO: Do we really  we need ProcessStatus and ProcessState - Do these DTO's do the same things?
 @JsonCodec case class ProcessStatus(status: StateStatus,
                                     deploymentId: Option[String],
                                     allowedActions: List[ProcessActionType],
+                                    customActions: List[ProcessStatus.CustomActionDTO],
```

Comment:
If we return static list from processManager, then we should return them in e.g. UIProcessObjects, so they are static also on FE.

Reply:
:+1: 
True.
https://github.com/TouK/nussknacker/pull/1367/commits/04ccf9b468a5bc8bd71786d1428ba42b2b362551